packages/tooltip/src/components/tooltip/Tooltip.tsx

```diff
@@ -1,14 +1,89 @@
-import { Text } from "@stenajs-webui/core";
+import { Indent, Row, Text } from "@stenajs-webui/core";
+import {
+  Icon,
+  stenaExclamationCircle,
+  stenaExclamationTriangle,
+  stenaInfoCircle,
+} from "@stenajs-webui/elements";
+import TippyComponent, {
+  TippyProps as TippyComponentProps,
+} from "@tippyjs/react";
+import cx from "classnames";
 import * as React from "react";
-import { Popover, PopoverProps, PopoverVariant } from "../popover/Popover";
+import { TippyCallbackRef } from "../../hooks/UseTippyInstance";
+import styles from "./Tooltip.module.css";
 
-export interface TooltipProps extends Partial<Omit<PopoverProps, "content">> {
-  variant?: PopoverVariant;
+type TooltipVariant = "info" | "warning" | "error";
+
+interface TippyProps
+  extends Partial<Omit<TippyComponentProps, "theme" | "render">> {
+  tippyRef?: TippyCallbackRef<HTMLDivElement>;
+}
+
+export interface TooltipProps extends TippyProps {
+  variant?: TooltipVariant;
   label: string;
+  multiRow?: boolean;
 }
 
-export const Tooltip: React.FC<TooltipProps> = ({ label, ...popoverProps }) => {
+const variantIcons = {
+  info: stenaInfoCircle,
+  warning: stenaExclamationCircle,
+  error: stenaExclamationTriangle,
+};
+
+const textComponent = (label: string, maxWidth: number) => (
```

Comment:
This is basically a React component. Make it one for real. (Or throw it away if the padding solution requires it.)

```suggestion
const TextComponent: React.FC<{label: string}> = ({label}) => (
```